### PR TITLE
style: allow Testimonial card to be full width with new prop

### PR DESCRIPTION
**Pull request by natalieweesh**
## Allow Testimonial Card to be full width

For the new User Toolkit tab of the dashboard we want the testimonial card to expand to the full width of the window, so added an optional `fullWidth` prop to Testimonial component and when it is true, it expands the max-width of the card to be 100% for large cards

Before: (it had max-width of 75rem)
Screen Shot 2020-05-08 at 2 31 16 PM

After: (max-width of 100%)
Screen Shot 2020-05-08 at 2 31 42 PM

---

## Merging your changes

When you are ready to merge to master and publish your changes, use the "squash and merge" button in GitHub, and follow the [PR Title Guide](https://github.com/Codecademy/client-modules#pr-title-guide) to format your PR title and write your PR merge description.

**IMPORTANT:** If your PR contains breaking changes, please remember to follow the instructions for breaking changes!

### CODE REVIEW

File: packages/brand-components/src/Testimonial/index.tsx [hunk 1]
```diff
@@ -17,13 +17,15 @@ export type Testimonial = {
 type TestimonialProps = {
   testimonial: Testimonial;
   size: 'small' | 'medium' | 'large';
+  fullWidth?: boolean;
```
Comment (JoshuaKGoldberg) [inline packages/brand-components/src/Testimonial/index.tsx:20]: What do you all think about doing this instead?

```suggestion
  size: 'small' | 'medium' | 'large' | 'full';
```

Fewer props -> simpler code -> happier devs

Comment (natalieweesh) [reply packages/brand-components/src/Testimonial/index.tsx:20]: 🆗  good idea

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@1.5.11-alpha.929c11.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5eb5ac6263bb1c4ec24c2e56--codecademy-styleguide.netlify.app>https://5eb5ac6263bb1c4ec24c2e56--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5eb5ac6263bb1c4ec24c2e56>Deploy Logs</a></p>

File: packages/brand-components/src/Testimonial/index.tsx [hunk 2]
```diff
@@ -49,7 +49,9 @@ export const Testimonial: React.FC<TestimonialProps> = ({
                 src={imageUrl}
                 alt={`Photo of ${firstName} ${lastName}`}
                 theme={theme}
-                className={cx({ [s.largeContainerAvatar]: size === 'large' })}
+                className={cx({
+                  [s.largeContainerAvatar]: size === 'large' || size === 'full',
+                })}
```
Comment (JoshuaKGoldberg) [inline packages/brand-components/src/Testimonial/index.tsx:54]: Oh I forgot about these... @rogervera do you remember why we don't just do `s[size]`? I feel like we could move all this logic to the styles, but I also forget why we didn't already 😄

File: packages/brand-components/src/Testimonial/styles.module.scss [hunk 3]
```diff
@@ -102,7 +105,8 @@ $background-pattern-offset: px-rem(16px);
 }
 
 .mediumContainer,
-.largeContainer {
+.largeContainer,
+.fullContainer {
   max-width: px-rem(708px);
```
Comment (rogervera) [inline packages/brand-components/src/Testimonial/styles.module.scss:110]: Hey @natalieweesh, with our conversation with Tímea, how do you feel about just changing this `max-width to 100%`, and not needing the new full size?

**Comment by natalieweesh** [comment]
@rogervera 👍 updated

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://5ebb0721667725db6361d0d5--codecademy-styleguide.netlify.app>https://5ebb0721667725db6361d0d5--codecademy-styleguide.netlify.app</a></p><p><a href=https://app.netlify.com/sites/codecademy-styleguide/deploys/5ebb0721667725db6361d0d5>Deploy Logs</a></p>

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/brand-components@1.5.14-alpha.7a56d7.0<br/></pre>

**Pull request merged**